src/data/valid/Database-metagenome-assembly.yaml

```diff
@@ -1,7 +1,8 @@
 metagenome_assembly_set:
   - id: nmdc:wfmgas-99-B7Vogx.1
     name: Metagenome assembly 1472_51277
-    was_informed_by: nmdc:omprc-11-547rwq94
+    was_informed_by: nmdc:omprc-11-547rwq94 # MAM 2024-05-22
+    #    was_informed_by: nmdc:act-11-547rwq94.1 # should this allow an omics processing as the informer? what about act? that's abstract isn't it?
```

Comment:
They way this slot is being used in practice is to tie WorkflowExecutionActivity subclass instances to OmicsProcessing & this is what the data portal uses The instances themselves are linked by has_input and has_output. We shouldn't make any further changes to this now b/c WorkflowChain in Berkeley schema addresses this.